server/services/bookerService.ts

```diff
@@ -29,11 +28,14 @@ export type Prisoner = {
   convictedStatus?: ConvictedStatus
 }
 
-export interface Visitor extends VisitorInfoDto {
+export type Visitor = {
```

Comment:
Fully define `Visitor` here instead of extending the API's type because we flatten the contents of `visitorRestrictions: []` to `banned` and `banExpiryDate` to make them easier to access.